lib/util/config-types.ts

```diff
@@ -68,9 +68,15 @@ interface ApiKeyConfig {
 
 export type BugsnagConfig = ApiKeyConfig
 export type MapillaryConfig = ApiKeyConfig
+export type NearbyFilterConfig = {
+  cardType: 'Stop' | 'RentalVehicle' | 'VehicleParking' | 'BikeRentalStation'
+  defaultOn: boolean
+  iconName: string
+}
 
 export type NearbyViewConfig = {
   alwaysShowLongName?: boolean
+  filterConfig?: Array<NearbyFilterConfig>
```

Comment:
```suggestion
  filters?: Array<NearbyFilterConfig>
```
Putting the word config inside a config